tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -90,6 +90,22 @@ static_assert(is_trivially_copyable_v<span<const int>::iterator>);
 static_assert(is_trivially_copyable_v<span<const int, 3>>);
 static_assert(is_trivially_copyable_v<span<const int, 3>::iterator>);
 
+// For portability, our implementation provides an additional guarantee beyond the Standard
+// that span and its iterators are standard-layout.
+static_assert(is_standard_layout_v<span<int>>);
+static_assert(is_standard_layout_v<span<int>::iterator>);
+static_assert(is_standard_layout_v<span<int, 3>>);
+static_assert(is_standard_layout_v<span<int, 3>::iterator>);
+static_assert(is_standard_layout_v<span<const int>>);
+static_assert(is_standard_layout_v<span<const int>::iterator>);
+static_assert(is_standard_layout_v<span<const int, 3>>);
+static_assert(is_standard_layout_v<span<const int, 3>::iterator>);
+
+// For performance, our implementation provides an additional guarantee beyond the Standard
+// that fixed size span has only the size of a pointer.

```

Comment:
"fixed-size" should be hyphenated, as it's a compound adjective.

Reply:
thats correct